Kitodo-DataAccess/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.kitodo</groupId>
+        <artifactId>kitodo-production</artifactId>
+        <version>3.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>kitodo-data-access</artifactId>
+    <name>Kitodo - Data Access</name>
+    <url>https://github.com/kitodo/kitodo-production</url>
+    <organization>
+        <name>kitodo.org</name>
+        <url>http://kitodo.org</url>
+    </organization>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.jena</groupId>
+            <artifactId>apache-jena-libs</artifactId>
```

Comment:
This artefact depends on `slf4j` but our logging framework is `log4j2`. `slf4j` must be excluded and the log4j2 bridge `log4j-slf4j-impl` must be added beside a minimal logging configuration.

Reply:
slf4j cannot be excluded. Apache Jena needs it. For tests, the non-operational bridge has been included (which should be ok for tests. If the tests fail, it is quite sure a problem in our code, not in Jena). For other uses, the module that will depend on this module has to include the appropriate bridge.

Reply:
It can as we did this already to remove a second logging framework. Jena needs only a implementation of slf4j for its own logging. This implementation could be provided by the slf4j package (which is delivered automatically as a Maven dependency) or by `log4j-slf4j-impl` package.